src/main/java/org/broadinstitute/consent/http/service/InstitutionService.java

```
@@ -55,23 +45,12 @@ public Institution updateInstitutionById(Institution institutionPayload, Integer
     isInstitutionNull(targetInstitution);
     checkUserId(userId);
     checkForEmptyName(institutionPayload);
-    Date updateDate = new Date();
-    institutionDAO.updateInstitutionById(
-        id,
-        institutionPayload.getName(),
-        institutionPayload.getItDirectorEmail(),
-        institutionPayload.getItDirectorName(),
-        institutionPayload.getInstitutionUrl(),
-        institutionPayload.getDunsNumber(),
-        institutionPayload.getOrgChartUrl(),
-        institutionPayload.getVerificationUrl(),
-        institutionPayload.getVerificationFilename(),
-        (Objects.nonNull(institutionPayload.getOrganizationType())
-            ? institutionPayload.getOrganizationType().getValue() : null),
-        userId,
-        updateDate
-    );
-    return institutionDAO.findInstitutionById(id);
+    try {
```

If you let this throw won't the resource layer wrap the error appropriately?

---

It should, yes. Will refactor.